rama-http/src/headers/x_robots_tag/element.rs

```diff
@@ -0,0 +1,46 @@
+use crate::headers::x_robots_tag::rule::Rule;
+use rama_core::error::OpaqueError;
+use std::fmt::Formatter;
+use std::str::FromStr;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Element {
+    bot_name: Option<String>, // or `rama_ua::UserAgent` ???
```

Comment:
You can copy https://github.com/hyperium/headers/blob/master/src/util/value_string.rs to https://github.com/plabayo/rama/tree/main/rama-http/src/headers/util and use that one here, that's also what the UserAgent typed header uses: https://docs.rs/headers/0.4.0/src/headers/common/user_agent.rs.html#43